# HG changeset patch
# User Rik <rik@octave.org>
# Date 1533227984 25200
#      Thu Aug 02 09:39:44 2018 -0700
# Node ID a0923a7878e33a6ad1d58cadf520af1daf018dc5
# Parent  a6c796206e7d8858306d17c39726fbfb87e61335
Update legacy function docstrings.

* isstr.m, setstr.m: Use @code{} around functions and add @seealso link to
replacement function.  Wrap comments to 80 characters rather than 72.

diff --git a/scripts/legacy/isstr.m b/scripts/legacy/isstr.m
--- a/scripts/legacy/isstr.m
+++ b/scripts/legacy/isstr.m
@@ -12,26 +12,26 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 ## GNU General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <https://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {} {} isstr (@var{a})
-## This function is obsolete.  Use ischar instead.
+## @deftypefn {} {} isstr (@var{x})
+## This function is obsolete.  Use @code{ischar} instead.
+## @seealso{ischar}
 ## @end deftypefn
 
 ## Author: jwe
 
-## At one time, Matlab docs stated that this function is obsolete and
-## would be removed in some future version.  Now users are told that it
-## should be avoided, but there is no mention of possible future
-## removal.
+## At one time, Matlab docs stated that this function is obsolete and would be
+## removed in some future version.  Now users are told that it should be
+## avoided, but there is no mention of possible future removal.
 
 function retval = isstr (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:legacy-function",
              "isstr is obsolete; please use ischar instead");
diff --git a/scripts/legacy/setstr.m b/scripts/legacy/setstr.m
--- a/scripts/legacy/setstr.m
+++ b/scripts/legacy/setstr.m
@@ -12,18 +12,19 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {Function File} {} setstr (@var{s})
-## This function is obsolete.  Use char instead.
+## @deftypefn {} {@var{s} =} setstr (@var{x})
+## This function is obsolete.  Use @code{char} instead.
+## @seealso{char}
 ## @end deftypefn
 
 ## Author: jwe
 
 ## At one time, Matlab docs stated that this function is obsolete and would be
 ## removed in some future version.  Now users are told that it should be
 ## avoided, but there is no mention of possible future removal.
 
